web/src/pages/authorized/breadcrumb/Breadcrumbs.tsx

```diff
@@ -75,7 +75,7 @@ export const Breadcrumbs: FC = () => {
 		} else {
 			const tokens = pathname.split('/');
 			setBreadcrumbInfo(
-				tokens.map(
+				tokens.filter((_e, i) => i < 3).map(
```

Comment:
I still see line 37-51 in this file that has Infra and Apps menus... Can you please remove the code that's not used anymore? Also, we should remove the Apps page.